manifests/v1beta1/components/controller/rbac.yaml

```diff
@@ -32,7 +51,10 @@ rules:
       - roles
       - rolebindings
     verbs:
-      - "*"
+      - "get"
+      - "create"
+      - "list"
+      - "watch"
   - apiGroups:
       - batch
     resources:
```

Comment:
@johnugeorge Should we limit scope for `deployment`, `batch`, and `kubeflow.org jobs` ?